src/renderers/controls/text.control.tsx

```diff
@@ -23,7 +23,8 @@ export class TextControl extends Control<ControlProps, ControlState> {
     const isValid = errors.length === 0;
     const divClassNames = 'validation' + (isValid ? '' : ' validation_error');
     const controlElement = uischema as ControlElement;
-    const maxLength = resolveSchema(schema, controlElement.scope.$ref).maxLength;
+    const resolvedSchema = resolveSchema(schema, controlElement.scope.$ref);
+    const maxLength = resolvedSchema ? resolvedSchema.maxLength : undefined;
```

Comment:
@eneufeld @edgarmueller please don't merge yet because I found a bug in here I have to fix first.
For nested properties like the one below, the code doesn't work, because `resolveSchema` descends down to the first `properties` object, but not the nested ones within that.

`"address": {`
&nbsp;&nbsp;`"type": "object",`
&nbsp;&nbsp;`"properties": {`
&nbsp;&nbsp;&nbsp;&nbsp;`"streetnumber": {`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`"type": "string",`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`"maxLength": 4`
&nbsp;&nbsp;&nbsp;&nbsp;`}`
&nbsp;&nbsp;`}`
`},`